debugging

```diff
@@ -0,0 +1,511 @@
+android: I/flutter :                                                            └Center(renderObject: RenderPositionedBox)
+android: I/flutter :                                                              └FlatButton(dirty; state: _FlatButtonState(859098856))
+android: I/flutter :                                                                └Container(BoxConstraints(88.0<=w<=Infinity, h=36.0); margin: EdgeDims(8.0, 8.0, 8.0, 8.0))
+android: I/flutter :                                                                  └Padding(renderObject: RenderPadding relayoutSubtreeRoot=up1)
+android: I/flutter :                                                                    └ConstrainedBox(BoxConstraints(88.0<=w<=Infinity, h=36.0); renderObject: RenderConstrainedBox relayoutSubtreeRoot=up2)
+android: I/flutter :                                                                      └DefaultTextStyle(inherit: false; color: Color(0xdd000000); size: 14.0; weight: 500; baseline: alphabetic)
+android: I/flutter :                                                                        └InkWell(state: _InkResponseState<InkResponse>(365061416))
+android: I/flutter :                                                                          └GestureDetector(state: _GestureDetectorState(383331210; gestures: tap; behavior: opaque))
+android: I/flutter :                                                                            └Listener(listeners: down; behavior: opaque; renderObject: RenderPointerListener relayoutSubtreeRoot=up3)
+android: I/flutter :                                                                              └Container(padding: EdgeDims(0.0, 8.0, 0.0, 8.0))
+android: I/flutter :                                                                                └Padding(renderObject: RenderPadding relayoutSubtreeRoot=up4)
+android: I/flutter :                                                                                  └Center(renderObject: RenderPositionedBox relayoutSubtreeRoot=up5)
+android: I/flutter :                                                                                    └Text("Dump App")
+android: I/flutter :                                                                                      └RawText(renderObject: RenderParagraph relayoutSubtreeRoot=up6)
+```
+
+This is the "flattened" tree, showing all the widgets projected
+through their various build functions. You'll see a lot of widgets in
+there that don't appear in your application's source, because they are
+inserted by the framework's widgets' build functions. For example,
+"InkFeatures" is an implementation detail of the "Material" widget.
+
+Since the `debugDumpApp()` call is invoked when the button changes
+from being pressed to being released, it coincides with the
+`FlatButton` object calling `setState()` and thus marking itself
```

Comment:
You're consistent about whether its `FlatButton` or "Material".  I'd opt for `Material`.